assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -97,6 +98,15 @@ function useUploadVideoFrame({ updateMediaElement }) {
     }
   };
 
+  /**
+   * Helper function get the file name from url.
+   *
+   * @param url
+   * @return {string}
+   */
+  const getFileName = (url) =>
```

Comment:
There's already `getFileName.js` from `edit-story/utils` that could perhaps be used.

It doesn't have any tests (or actual usage 🤔 ) though, so not sure it works in this context.

Reply:
I tried using it. But it returned with the ext. So `filename.mp4`. This function just gives the filename without the ext.